### ISSUE TITLE
Tooltip is not provided for 'next/ previous' icons

### ISSUE THREAD

**Issue by robertmausser** [adaptlearning/adapt-contrib-narrative#274]
### Subject of the issue/enhancement/features

Tooltip is not provided for 'next/ previous' icons.

### Your environment
* 5.31.10
* all browsers
* all OSes

### Steps to reproduce

Use screen reader to move to previous/next buttons.

### Expected behaviour

Tooltip should be provided for previous/next icons.

### Actual behaviour

Tooltip is not provided for previous/next icons.

### Screenshots (if you can)

https://app.screencast.com/JaMrV2j8R3oIB

### How to Fix

"1. Create a <div> element, give it a role=""tooltip"" and an id=""IDREF"". Then, give it a brief description of the action.
2. This element has to be accessible by focusing as well as hovering over. Both functionalities can be provided using CSS.
3. The tooltip has to be visually hidden using CSS.
4. Associate the tooltip with its button using aria-labelledby.
5. After applying this keyboard-accessible tooltip, consider removing the title attribute from the element.

Refer to:
https://inclusive-components.design/tooltips-toggletips/
https://a11yproject.com/posts/how-to-hide-content/"

**Comment by kirsty-hames** [comment]
For consistency with other Adapt buttons we could use the [core tooltip API](https://github.com/adaptlearning/adapt-contrib-core/pull/207) for this.
